src/HomePage/CategoryShopping/Category.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useContext, useEffect, useState } from 'react';
+import './Category.scss';
+import { DFS } from '../../Additional/additional_api';
+
+export const Category = () => {
+  const [serverData, setServerData] = useState([]);
+  const [mobiles, setMobiles] = useState([]);
```

Comment:
what's the purpose to store the whole array if you just need a length?

Reply:
resolved